### ISSUE TITLE
How to change the joint limit of cobotta arm

### ISSUE THREAD

**Issue by zhoubo2018** [DENSORobot/denso_robot_ros#16]
I try to write moveit python file to control the arm, I write several points, sometimes the moveit plan is successful, but when the arm moves, it will reach the joint limit, the arm led will turn  yellow, I think it may be a joint limit setting in control box, so I want to ask you how to set that. Looking forward to your reply! Thank you very much.

**Comment by ToshitakaSuzuki** [comment]
The joint angle limits are specified in URDF file.
But why are you trying to change the joint angle limits?
Can't you change your target points?

**Comment by zhoubo2018** [comment]
yeah, because I use the windows/android app to move the arm to the position which I want to move in ros, it worked, so there might be some limit in ros or in b-cap

**Comment by ToshitakaSuzuki** [comment]
> yeah, because I use the windows/android app to move the arm to the position which I want to move in ros

OK, I understand your requirement.

The problem might be that moveit generates trajectories that joint angle limits are not taken into account. But I don't why...

**Comment by AlanSunAlan** [comment]
I think there might be necessary to change the joint limit in urdf. I found that the joint limits in the original urdf file are just too large. You can first determine the real limit value by moving the joints to the limit point by hand, then record all the values in order to change the urdf.
